decidim-accountability/spec/shared/manage_timeline_examples.rb

```diff
@@ -8,7 +8,7 @@
     click_on "Edit", match: :first
 
     within ".edit_timeline_entry" do
-      fill_in :timeline_entry_entry_date, with: Date.current.strftime("%d/%m/%Y")
+      fill_in :timeline_entry_entry_date, with: Date.current.strftime("%m/%d/%Y")
```

Comment:
This isn't done in v0.29 nor develop. Any reason why we need this? Maybe we need to add a comment if that's the case?

Reply:
For some reason just the pipeline in 0.28 is failing with this error. Maybe is something related to Rspec / Capybara.

Reply:
you can see that https://github.com/decidim/decidim/actions/runs/12936222967/job/36091021551?pr=13915  it actually failed 4 times with the same error.